pepper-import/src/main/java/bio/terra/pearl/pepper/ActivityImporter.java

```diff
@@ -615,20 +615,20 @@ SurveyType getSurveyType(FormActivityDef activityDef) {
      * After testing some more surveys, we might want to upgrade this to use a html->markdown parsing library */
     static Map<String, String> JUNIPER_PEPPER_STRING_MAP = Map.ofEntries(
             Map.entry("\\$ddp.participantFirstName\\(\\)", "{profile.givenName}"),
-            Map.entry("\\<p.*?\\>", ""),
-            Map.entry("\\</p\\>", "\\\\n"),
-            Map.entry("\\<span.*?\\>", ""),
-            Map.entry("\\</span\\>", ""),
-            Map.entry("\\<b.*?\\>", "**"),
-            Map.entry("\\</b\\>", "**"),
-            Map.entry("<b.*?>", "**"),
-            Map.entry("</b>", "**"),
-            Map.entry("\\<i.*?\\>", "*"),
-            Map.entry("\\</i\\>", "*"),
-            Map.entry("<i.*?>", "*"),
-            Map.entry("</i>", "*"),
-            Map.entry("\\<em.*?\\>", "*"),
-            Map.entry("\\</em\\>", "*"),
+//            Map.entry("\\<p.*?\\>", ""),
+//            Map.entry("\\</p\\>", "\\\\n"),
+//            Map.entry("\\<span.*?\\>", ""),
+//            Map.entry("\\</span\\>", ""),
+//            Map.entry("\\<b.*?\\>", "**"),
+//            Map.entry("\\</b\\>", "**"),
+//            Map.entry("<b.*?>", "**"),
+//            Map.entry("</b>", "**"),
+//            Map.entry("\\<i.*?\\>", "*"),
+//            Map.entry("\\</i\\>", "*"),
+//            Map.entry("<i.*?>", "*"),
+//            Map.entry("</i>", "*"),
+//            Map.entry("\\<em.*?\\>", "*"),
+//            Map.entry("\\</em\\>", "*"),
```

Comment:
Somtimes the HTML escaping saves a lot of trouble, so I'm gonna keep it here, but I find it often hurts more than it helps...

Reply:
cool, can you add a comment briefly explaining?